### PR TITLE
Update LSE-63.tex

### PR DESCRIPTION
**Pull request by tonytyson**
Tim,
I just made edits throughout, but mainly in the last 2 sections.
Tony

### CODE REVIEW

File: LSE-63.tex [hunk 1]
```diff
@@ -25,6 +25,8 @@
 \addtohist{5}{2011-07-10}{General edits}{Tony Tyson}
 \addtohist{6}{2011-07-15}{Added content}{Tony Tyson}
 \addtohist{7}{2011-07-31}{Added content}{Tony Tyson}
+\addtohist{8}{2017-04-13}{Added content}{Tim Jenness}
+\addtohist{7}{2017-04-16}{Added content}{Tony Tyson}
```
Comment (timj) [inline LSE-63.tex:29]: I think the new LPM-51 policy is to only increment the release number at change control time. I can fix that up before merging.

**Comment by timj** [comment]
@tonytyson I've added you to this repo so you can make branches here. I'll close this pull request and migrate your branch here so that I can do some minor cleanups.

**Comment by tonytyson** [comment]
Thanks

**Comment by timj** [comment]
Reopened as #3

**Pull request closed without merging**